configs/11.0/packages/valgrind/sources

```diff
@@ -20,11 +20,11 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.12.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16456
```

Comment:
This revision is from after the 3.13 branch was created:

```
$ svn log --revision 16419
------------------------------------------------------------------------
r16419 | sewardj | 2017-05-24 13:10:30 -0300 (Wed, 24 May 2017) | 2 lines

Create branches/VALGRIND_3_13_BRANCH as a copy of trunk r16418.

------------------------------------------------------------------------
```

Besides that, this revision is also newer than the creation of the tag 3.13.0:

```
$ svn log --stop-on-copy tags/VALGRIND_3_13_0
------------------------------------------------------------------------
r16448 | sewardj | 2017-06-16 04:27:45 -0300 (Fri, 16 Jun 2017) | 2 lines

Swizzle external.

------------------------------------------------------------------------
r16447 | sewardj | 2017-06-16 04:26:24 -0300 (Fri, 16 Jun 2017) | 2 lines

Create tags/VALGRIND_3_13_0 as a copy of branches/VALGRIND_3_13_BRANCH, r16446.

------------------------------------------------------------------------
```

Did you really want to update that far, or were you planning to update only to 3.12?